libs/providers/langchain-google-webauth/tsdown.config.ts

```diff
@@ -0,0 +1,4 @@
+import { defineConfig } from "tsdown";
+import { getBuildConfig } from "@langchain/build";
+
+export default defineConfig([getBuildConfig()]);
```

Comment:
Are brackets required?

```suggestion
export default defineConfig(getBuildConfig());
```